components/kserve/kserve.go

```diff
@@ -184,9 +185,13 @@ func (k *Kserve) configureServerless(instance *dsciv1.DSCInitializationSpec) err
 	case operatorv1.Managed: // standard workflow to create CR
 		switch instance.ServiceMesh.ManagementState {
 		case operatorv1.Unmanaged, operatorv1.Removed:
-			return fmt.Errorf("ServiceMesh is need to set to 'Managaed' in DSCI CR, it is required by KServe serving field")
+			return fmt.Errorf("ServiceMesh is need to set to 'Managed' in DSCI CR, it is required by KServe serving field")
 		}
-		serverlessInitializer := feature.NewFeaturesInitializer(instance, k.configureServerlessFeatures)
+		origin := featurev1.Origin{
+			Type: featurev1.ComponentType,
+			Name: k.GetComponentName(),
```

Comment:
i have been thinking, if one day we have a FT CR which is shared by multiple components.  (similar to odh-model-controller deployed by both kserve and modelmesh) how should we set this Name

Reply:
That would mean that actual instance of the struct needs to be created in some common base for both. Or we use some suffix for the feature name.